server/controllers/userStudiesController/userStudiesController.test.js

```diff
@@ -0,0 +1,88 @@
+import userStudiesController from '.'
+import {
+  createResponse,
+  createRequestWithUser,
+  createUser,
+  createStudyTableRowData,
+} from '../../../test/fixtures'
+import { createStudies } from '../../../test/testUtils'
+import { collections } from '../../utils/mongoCollections'
+
+describe('userStudiesController', () => {
+  describe(userStudiesController.index, () => {
+    describe('When successful', () => {
+      let appDb
+      let user
+
+      beforeAll(async () => {
+        user = createUser({
+          uid: 'owl',
+          preferences: {},
+          access: ['YA', 'LA', 'MA'],
+        })
+
+        appDb = await global.MONGO_INSTANCE.db('studies')
+
+        await appDb.collection(collections.users).insertOne(user)
+        await appDb.collection(collections.metadata).insertMany(createStudies())
+      })
+      afterAll(async () => {
+        await appDb.dropDatabase()
+      })
+
+      it('returns a list of studies with the column properties to be displayed', async () => {
+        const response = createResponse()
+        const request = createRequestWithUser(
+          {
+            query: { sort: {} },
```

Comment:
we should have a test that does not have a query at all to expose the bug caused by not having sort params and then resolve it.